### PR TITLE
248 session expires

### LINKED ISSUE #189: Session expires very often
At least once a day (and perhaps more too, I don't really know), when I enter the platform my session is expired and I have to "Login with GitHub" again.

### PR DESCRIPTION
**Pull request by aguspina**
Fixes #189

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -182,6 +190,13 @@
   add_index "paths", ["category_id"], name: "index_paths_on_category_id", using: :btree
   add_index "paths", ["language_id"], name: "index_paths_on_language_id", using: :btree
 
+  create_table "relationships", force: true do |t|
+    t.integer  "follower_id"
+    t.integer  "followed_id"
+    t.datetime "created_at"
```
Comment (flbulgarelli) [inline db/schema.rb]: Please check you db and migrations. Here yo have introduced several changes that should not be there.

**Comment by flbulgarelli** [comment]
This this somewhat hacky to me. Do you have some link or explanation for this fix?

**Comment by flbulgarelli** [comment]
Oh, and please fix tests too :smile:

**Comment by aguspina** [comment]
By now the session expires fix is correct, but I couldn't fix those specs. @flbulgarelli any idea?

**Comment by flbulgarelli** [comment]
@aguspina Tests are broken since by some reason you are being always logged in, and you can not be logged out. 

Simply verify that by starting rails server, and trying to logout - it will  not work. 

Please revise that logic.

**Comment by flbulgarelli** [comment]
@aguspina bump

**Comment by flbulgarelli** [comment]
Oh, and branch has been rebased here: https://github.com/mumuki/mumuki-platform/tree/aguspina-248-session-expires

**Comment by flbulgarelli** [comment]
Closing and creating new PR #320

**Pull request closed without merging**